main.go

```diff
@@ -1,32 +1,24 @@
 package main
 
 import (
-	"fmt"
 	"log"
-
-	_ "github.com/lib/pq"
-	"gorm.io/driver/postgres"
-	"gorm.io/gorm"
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/mkrs2404/eKYC/api/services"
+	"github.com/mkrs2404/eKYC/database"
+	"github.com/mkrs2404/eKYC/server"
+	_ "github.com/mkrs2404/eKYC/server"
+	"gorm.io/gorm/logger"
 )
 
 func main() {
-
-	connection_string := "host=localhost user=postgres password=postgres port=5432 sslmode=disable"
-	db, err := gorm.Open(postgres.New(postgres.Config{DSN: connection_string}), &gorm.Config{})
-	handleError(err)
-
-	database, err := db.DB()
-	handleError(err)
-
-	err = database.Ping()
-	handleError(err)
-
-	fmt.Println("Connected to Postgres")
-
-}
-
-func handleError(err error) {
+	err := godotenv.Load()
 	if err != nil {
```

Comment:
No need for else block
```
if err != nil {
   log.Fatal("Error fetching")
}

<your logic here>
```
Read https://en.wikipedia.org/wiki/Guard_(computer_science)

Reply:
Resolved